cli/azd/cmd/root.go

```diff
@@ -414,6 +415,35 @@ func NewRootCmd(
 		panic(err)
 	}
 
+	if err := rootContainer.Invoke(func(alphaFeatureManager *alpha.FeatureManager) error {
+		llmEnabledError := llm.IsLlmFeatureEnabled(alphaFeatureManager)
+		if llmEnabledError != nil {
+			root.Add("mcp", &actions.ActionDescriptorOptions{
+				Command: &cobra.Command{
+					RunE: func(cmd *cobra.Command, args []string) error {
+						return llmEnabledError
+					},
+				},
+			})
+		} else {
+			root.Add("mcp", &actions.ActionDescriptorOptions{
+				Command:        newMcpCmd(),
+				FlagsResolver:  newMcpFlags,
+				ActionResolver: newMcpAction,
+				HelpOptions: actions.ActionHelpOptions{
+					Description: getCmdMcpHelpDescription,
+					Footer:      getCmdMcpHelpFooter,
+				},
```

Comment:
I think we may not need this and can remove `getCmdMcpHelpDescription` and `getCmdMcpHelpFooter` since we don't have extra description or footer content, like "add" command for example